offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -743,9 +858,151 @@ export class CkbHandler {
+      return null;
+    }
+    const recipientOutput = nonNullable(tx.outputs[0]);
+    const recipientCapacity = nonNullable(recipientOutput.capacity);
+    const recipientTypescript = recipientOutput.type;
+    const recipientLockscript = nonNullable(recipientOutput.lock);
+    logger.debug('recipientTypescript:', recipientTypescript);
+
+    const sudtTypescript = ForceBridgeCore.config.ckb.deps.sudtType.script;
+    const previousOutput = nonNullable(tx.inputs[0].previousOutput);
+    const preHash = previousOutput.txHash;
+    const txPrevious = await this.ckb.rpc.getTransaction(preHash);
+    if (txPrevious == null) {
+      return null;
+    }
+    const senderLockscript = txPrevious.transaction.outputs[Number(previousOutput.index)].lock;
+    const committeeMultisigLockscript = parseAddress(getOmniLockMultisigAddress());
+    if (
+      !senderLockscript ||
+      (senderLockscript.codeHash === committeeMultisigLockscript.code_hash &&
+        senderLockscript.hashType === committeeMultisigLockscript.hash_type &&
+        senderLockscript.args === committeeMultisigLockscript.args)
+    ) {
+      logger.error(`sender should not be committee multisig cell`);
+      return null;
+    }
+
+    if (tx.inputs.length > 1) {
+      for (let i = 1; i < tx.inputs.length; i++) {
+        const thisPreviousOutput = nonNullable(tx.inputs[i].previousOutput);
+        const thisPreHash = thisPreviousOutput.txHash;
+        const thisTxPrevious = await this.ckb.rpc.getTransaction(thisPreHash);
+        if (thisTxPrevious == null) {
+          return null;
+        }
+        const thisSenderLockscript = thisTxPrevious.transaction.outputs[Number(thisPreviousOutput.index)].lock;
+        if (
+          !thisSenderLockscript ||
+          thisSenderLockscript.codeHash !== senderLockscript.codeHash ||
+          thisSenderLockscript.hashType !== senderLockscript.hashType ||
+          thisSenderLockscript.args !== senderLockscript.args
+        ) {
+          logger.error(`inputs contain different lockscripts`);
+          return null;
+        }
+      }
+    }
+
+    if (
+      recipientLockscript.codeHash !== committeeMultisigLockscript.code_hash ||
+      recipientLockscript.hashType !== committeeMultisigLockscript.hash_type ||
+      recipientLockscript.args !== committeeMultisigLockscript.args
+    ) {
+      logger.error(`the first output must be committee multisig`);
+      return null;
+    }
+
+    if (tx.outputs.length > 1) {
+      for (let i = 1; i < tx.outputs.length; i++) {
+        const thisLockscript = nonNullable(tx.outputs[i].lock);
+        if (
+          thisLockscript.codeHash === committeeMultisigLockscript.code_hash &&
+          thisLockscript.hashType === committeeMultisigLockscript.hash_type &&
+          thisLockscript.args === committeeMultisigLockscript.args
+        ) {
+          logger.error(`the outputs except first must not be committee multisig`);
```

Comment:
This is not a error, it is just a tx that is not what we want(a validated lock tx)
```suggestion
          logger.warn(`the outputs except first must not be committee multisig`);
```